packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -315,15 +300,15 @@ const CFDPersonalDetailsForm = ({
                                 >
                                     <Text
                                         as='p'
-                                        size={is_in_personal_details_modal ? 'xxs' : 'xxxs'}
+                                        size='xxs'
                                         align='center'
                                         className='details-form__description'
                                         data-testid='dt_cfd_details_form_description'
                                         style={
                                             isMobile()
                                                 ? {
                                                       textAlign: 'center',
-                                                      fontSize: is_in_personal_details_modal ? '1.2rem' : '1rem',
+                                                      fontSize: '1.2rem',
                                                       color: '#333333',
                                                   }
                                                 : {}
```

Comment:
[Question] Can we use `Text` props for these css variables?

Reply:
Yes. I have updated the code